lib/bundle/brew_dumper.rb

```diff
@@ -59,12 +55,8 @@ def self.formula_aliases
     private
 
     def self.formulae_info
-      begin
-        installed_formulae = JSON.load(`brew info --json=v1 --installed`) || []
-        installed_formulae.map { |info| formula_inspector info }
-      rescue JSON::ParserError
-        []
-      end
+      require "cmd/info"
+      Homebrew.formulae_json.map { |info| formula_inspector info }
```

Comment:
Sorry, haven't pushed that to the other PR yet. It may be able to be simplified further into using `Formula.hash` after I finish the outdated refactoring.